Packs/GoogleCloudSCC/ReleaseNotes/1_1_0.md

```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+##### Google Cloud SCC
+- Added google-cloud-cai-asset-resource-list and google-cloud-cai-asset-owner-get commands.
```

Comment:
Please update the commands names here as well

Reply:
done